.gitignore

```diff
@@ -1,9 +1,7 @@
 *.test
-/bin
+/.tmp/
 
 clientcompat/pycompat/ENV
 *.pyc
 
-build
```

Comment:
This is used in generating the documentation website, and should not be removed.